USAGE.md

```diff
@@ -0,0 +1,192 @@
+
+```ruby
+pod 'MyFramework', :path => '../../MyFrameWork/'
+```
+
+## Handle versioning as you would with any other pod
+- If you don't understand Cocoapods' concept of 'semantic versioning', watch [this video](https://www.youtube.com/watch?v=x4ARXyovvPc)
+- Increment `version` in your podspec
+
+```ruby
+s.version = '0.0.1'
+```
+- Make sure your framework distribution repo's podspec's source attribute looks something like this:
+
+````ruby
+s.source           = {
+  :git => "https://github.com/CompanyName/RepoName.git", 
+  :tag => s.version.to_s
+}
+```` 
+
+- Connect your local framework distribution repo to a remote repo. Github walks you through this, but assuming you've made a repo on github, do this:
+
+```ruby
+git remote add origin https://github.com/CompanyName/RepoName.git
+git push -u origin master
+```
+- Commit and push to make sure your actual code is in your remote repo before you tag it locally and push the tags.
+
+```ruby
+git commit -A 
+git push -u origin master
+```
+
+- Tag your local repo and push that tag to your git remote
+```ruby
+git tag 0.0.1 #(make sure this matches s.version in your podspec) 
+git push origin --tags
+```
+- If you mess up tagging or want to revert your tags for some reason, you can remove remote tags like this:
+
+```ruby
+git push --delete origin 0.0.1
+```
+
+and local tags like this:
+
+```ruby
+git tag -d 0.0.1
+```
+
+- If this is a private repo and you're working with it locally, don't forget to 
+
+```ruby
+pod repo update [YourRepoName]
+```
+
+and maybe
+
+```ruby
+pod repo push MyFramework MyFramework.podspec --allow-warnings
+```
+
+This last command will, as part of the push to your local repo, actually lint the new framework. If you ultimately want to submit it to the Cocoapods Specs repo, you'd need to lint it anyway. 
+
+## Test with remote repo
+Point an application at your new framework. If you've gotten this far (linted and packaged) it should definitely work!
+
+```ruby
+pod 'MyFramework', :git => 'https://github.com/CompanyName/RepoName.git', :tag => '0.0.1'
+```
+
+# Known Issues / Limitations
+Cocoapods-Packager can't produce a binary from a pod that contains Swift dependencies. See [here](https://github.com/CocoaPods/cocoapods-packager/issues/115) to track this issue. This limitation ends up causing a good deal of extra work. In my case, I was working with a single podspec that had mixed objective-c and Swift. To make use of the packager I had to refactor all of those swift dependencies into their own pods until the main library was objective-c only. 
+
+# Possible Gotchas
+**No headers created**
+This can happen if your spec does not lint. The packager may successfully complete and create your framework, but there will be no headers. Without headers, nothing will work, so...
+
+**The `source_files` pattern did not match any file**
+The packager creates a podspec with these lines:
+
+```ruby
+s.platform = :ios, '9.0' 
+s.ios.platform = :ios, '9.0'
```

Comment:
this line is a misuse of the podspec DSL, and the packager generating it is _definitely_ a bug